assets/wizards/engagement/views/reader-activation/index.js

```diff
@@ -74,56 +73,51 @@ export default withWizardScreen( () => {
 				description={ __( 'Configure a set of features for reader activation.', 'newspack' ) }
 				toggleChecked={ !! config.enabled }
 				toggleOnChange={ value => updateConfig( 'enabled', value ) }
-			/>
-			<Card noBorder>
-				<CheckboxControl
-					label={ __( 'Enable Sign In/Account link', 'newspack' ) }
-					help={ __(
-						'Display an account link in the site header. It will allow readers to register and access their account.',
-						'newspack'
-					) }
-					checked={ !! config.enabled_account_link }
-					onChange={ value => updateConfig( 'enabled_account_link', value ) }
-				/>
-				<TextControl
-					label={ __( 'Newsletter subscription text on registration', 'newspack' ) }
-					help={ __(
-						'The text to display while subscribing to newsletters on the registration modal.',
-						'newspack'
-					) }
-					value={ config.newsletters_label }
-					onChange={ value => updateConfig( 'newsletters_label', value ) }
-				/>
-				<Grid columns={ 2 } gutter={ 16 }>
-					<TextControl
-						label={ __( 'Terms & Conditions Text', 'newspack' ) }
-						help={ __( 'Terms and conditions text to display on registration.', 'newspack' ) }
-						value={ config.terms_text }
-						onChange={ value => updateConfig( 'terms_text', value ) }
-					/>
-					<TextControl
-						label={ __( 'Terms & Conditions URL', 'newspack' ) }
-						help={ __( 'URL to the page containing the terms and conditions.', 'newspack' ) }
-						value={ config.terms_url }
-						onChange={ value => updateConfig( 'terms_url', value ) }
-					/>
-				</Grid>
-			</Card>
-			{ isActiveCampaign && (
```

Comment:
Did you mean to remove the ActiveCampaign master list settings?

Reply:
The rebase clearly didn't go well 😅
I'll open a new PR, it'll be easier

Reply:
https://github.com/Automattic/newspack-plugin/pull/1852